plugins/woocommerce/client/admin/client/customize-store/index.tsx

```diff
@@ -1,481 +1,303 @@
+		recordEvent( 'customize_your_store_intro_browse_all_themes_click' );
+		markTaskComplete();
+	};
 
-	const taskReferral = getCYSTaskReferral();
+	const chevronIcon = isRTL() ? chevronRight : chevronLeft;
 
-	if ( taskReferral ) {
-		removeCYSTaskReferral();
-		window.location.href = taskReferral.returnUrl;
-	}
-};
+	const sidebarTitle = (
+		<Button href={ getNewPath( {}, '/', {} ) }>
+			{ __( 'Customize your store', 'woocommerce' ) }
+		</Button>
+	);
 
-export const machineActions = {
-	updateQueryStep,
-	redirectToWooHome,
-	redirectToThemes,
-	redirectToReferrer,
-	goBack,
-};
+	const sidebarDescription = __(
+		'Design a store that reflects your brand and business. Customize your active theme, select a professionally designed theme, or create a new look using our store designer.',
+		'woocommerce'
+	);
 
-export const customizeStoreStateMachineActions = {
-	...introActions,
-	...machineActions,
-};
+	return (
+		<div className="woocommerce-customize-store__container">
+			<div className="mobile-header">
+				<Button
+					className="mobile-header__back-button"
+					href={ getNewPath( {}, '/', {} ) }
+					icon={ chevronIcon }
+					label={ __( 'Back', 'woocommerce' ) }
+				/>
+				<h1 className="mobile-header__title">
+					{ __( 'Customize your store', 'woocommerce' ) }
+				</h1>
+			</div>
+			<div className="woocommerce-customize-store-sidebar">
+				<motion.div
+					className="woocommerce-edit-site-layout__header-container"
+					animate={ 'view' }
+				>
+					<SiteHub
+						variants={ {
+							view: { x: 0 },
+						} }
+						isTransparent={ false }
+						className="woocommerce-edit-site-layout__hub"
+					/>
+				</motion.div>
+
+				<VStack
+					className="woocommerce-edit-site-sidebar-navigation-screen__main"
+					spacing={ 0 }
+					justify="flex-start"
+				>
+					<HStack
+						spacing={ 4 }
+						alignment="flex-start"
+						className="woocommerce-edit-site-sidebar-navigation-screen__title-icon"
+					>
+						<SidebarButton
+							href={ getNewPath( {}, '/', {} ) }
+							icon={ chevronIcon }
+							label={ __( 'Back', 'woocommerce' ) }
+							showTooltip={ false }
+						/>
+
+						<Heading
+							className="woocommerce-edit-site-sidebar-navigation-screen__title"
+							level={ 1 }
+							as="h1"
+						>
+							{ sidebarTitle }
+						</Heading>
+					</HStack>
+
+					<div className="woocommerce-edit-site-sidebar-navigation-screen__content">
+						<p className="woocommerce-edit-site-sidebar-navigation-screen__description">
+							{ sidebarDescription }
+						</p>
+					</div>
+				</VStack>
+			</div>
 
-export const customizeStoreStateMachineServices = {
-	...introServices,
-	...transitionalServices,
-	browserPopstateHandler,
-	markTaskComplete,
+			<div className="woocommerce-customize-store-main">
+				<div className="woocommerce-customize-store-banner">
+					<div className="woocommerce-customize-store-banner-content">
+						<div className="banner-actions">
```

Comment:
Same with this class name.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/62158/commits/aab988703281390ab08369ff182fb59352522a77. The class is also used in LYS but I don't want to touch it in scope of this PR.